docs/02-app/02-api-reference/05-next-config-js/serverSourceMaps.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: serverSourceMaps
+description: Enables source maps generation for the server production bundle.
+---
+
+Since Next.js minifies the server production bundle to reduce the bundle size, source maps are also disabled by default. To enable the source maps generation, set `serverSourceMaps` to `true`.
+
+```js filename="next.config.js" switcher
+/** @type {import('next').NextConfig} */
+
+const nextConfig = {
+  serverSourceMaps: true,
+}
+
+module.exports = nextConfig
+```
+
+```js filename="next.config.mjs" switcher
```

Comment:
I'm not sure 3 code blocks will work as we only support JS and TS. Maybe you can update the filename for the first one to be filename="next.config.js|mjs"?